platform-modules/mimetype-manager/src/main/scala/org/sunbird/mimetype/mgr/impl/EcmlMimeTypeMgrImpl.scala

```diff
@@ -90,4 +95,30 @@ class EcmlMimeTypeMgrImpl(implicit ss: StorageService) extends BaseMimeTypeManag
 		}
 	}
 
+	override def review(objectId: String, node: Node)(implicit ec: ExecutionContext, ontologyEngineContext: OntologyEngineContext): Future[Map[String, AnyRef]] = {
+		validate(node)
+		Future(getEnrichedMetadata(node.getMetadata.getOrDefault("status", "").asInstanceOf[String]))
+	}
+
+	def validate(node: Node)(implicit ec: ExecutionContext, oec: OntologyEngineContext): Unit = {
+		val artifactUrl = node.getMetadata.getOrDefault("artifactUrl", "").asInstanceOf[String]
```

Comment:
Why don't you write a method in Node class to get artifactUrl?

Reply:
done